charts/virtual-kubelet/values.yaml

```diff
@@ -6,12 +6,18 @@ image:
 ## `provider` should be one of aws, azure, azurebatch, etc...
 provider: 
 nodeName: "virtual-kubelet"
-nodeTaint: "azure.com/aci"
 nodeOsType: "Linux"
 monitoredNamespace: ""
 apiserverCert: 
 apiserverKey: 
 
+taint:
+  enabled: true
+  key: virtual-kubelet.github.io/provider
```

Comment:
We could also do something like node.kubernetes.io/virtual-kubelet ? This would be in line with what k8s auto appleis on things.

Reply:
Not for long 😈 

Honestly it's probably fine for now, but probably shouldn't make a new release without changing it.

Reply:
> We could also do something like node.kubernetes.io/virtual-kubelet ? This would be in line with what k8s auto applies on things.

It might be better to go with a virtual-kubelet "owned" prefix instead for things like taints, annotations, etc.  That allows us to use keys that are nice and descriptive, like `provider`, without worrying about naming conflicts.

`virtual-kubelet.io` is 🔥

Reply:
I agree with your statement about descriptive name and not worrying about conflict. Let’s go with virtual-kubelet.io

I’ve gone ahead and purchased that domain. We plan to put a static site up with netlify or something, so let’s just use that.